static/bidder-params/intertech.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Intertech Adapter Params",
+    "description": "A schema which validates params accepted by the Intertech adapter",
+    "type": "object",
+    "properties": {
+      "page_id": {
+        "type": "integer",
+        "minLength": 1,
+        "description": "Special Page Id provided by Intertech Manager"
+      },
+      "imp_id": {
+        "type": "integer",
+        "minLength": 1,
+        "description": "Special identifier provided by Intertech Manager"
+      }
+    },
+    "required": [
+      "page_id",
+      "imp_id"
+    ]
+  }
```

Comment:
Placement id should be defined in here. Based on the comments in ExtImpIntertech, seems like it should be oneOf placementid or the combination of page_id and imp_id.